### PR TITLE
[Vol2-xdl]Add section on explainability of linear models.

### PR DESCRIPTION
**Pull request by cheungdaven**
*Description of changes:*

Add section on explainability of linear models.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by cheungdaven** [comment]
Hi @astonzhang this section is ready for review, please have a look if you are free. thanks!

**Comment by astonzhang** [comment]
Merging it for now so coauthors can work on it. Will revise later

File: chapter_explainability/linear.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+```{.python .input}
+from d2l import torch as d2l
+import numpy as np
+import pandas as pd
+from sklearn import linear_model
+from sklearn.metrics import mean_absolute_error
+```
+
+## Explaining Linear Models
+
```
Comment (astonzhang) [inline chapter_explainability/linear.md:32]: Don't leave space between a title and its subtitle empty. Can you add some text?

File: chapter_explainability/linear.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Explainability of Linear Models
+
+Linear models describe a target variable in terms of a linear combination of predictive variables. Models such as linear regression, lasso regression, logistic regression, and softmax regression are the popular variants of linear models :numref:`chap_linear`. The linearity assumption limits linear models' expressiveness but makes them easy-to-understand. Even though linear models are not perfect fit solutions oftentimes, they are extensively adopted in a multitude of domains especially when people are concerned more about what drives models' behaviors than predictive accuracy. Correctly interpreting linear models is a critical step to model understanding and is fundamental to more advanced explanation methods. 
+
+
+## Revisit Linear Models
```
Comment (astonzhang) [inline chapter_explainability/linear.md:6]: Revisiting

**Pull request merged**